apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -475,6 +476,56 @@ public async Task ReadStrongSingle()
         }
         // [END spanner_test_strong_read_concurrent_updates]
 
+        // [START spanner_test_exact_read_single_use_with_return_read_timestamp]
```

Comment:
Hmm, I see the region tag is `spanner_test` so maybe they should be shown as tests?

Reply:
I'm not sure these are meant to be samples honestly. What are the region tags here being used for?

Reply:
Oh I don't think the region tags are used for anything. These are not pulled into docs. I think we have them because someone started doing it this way initially and we just continued with it...

I can leave it as is for consistency and we can get rid of all them in one cleanup PR. Or I can remove it from here now. Let me know what you think might be better.

Reply:
I'ts fine to leave these in and then do a PR removing them.
But I think we should remove them, there's the SnippetTracker bot which is pulling these into the devrel services sample DB which means someone could include these on docs.
I'll create an issue so that we don't forget. Thanks!

Reply:
Thanks, I removed the tags from this PR since Lalji is already working on removing the tags from the rest of the tests.